src/controllers/reactWebviewBaseController.ts

```diff
@@ -105,18 +105,17 @@ export abstract class ReactWebviewBaseController<State, Reducers>
 
     protected _getHtmlTemplate() {
         const nonce = getNonce();
-        const baseUrl =
-            this._getWebview()
-                .asWebviewUri(
-                    vscode.Uri.joinPath(
-                        this._context.extensionUri,
-                        "out",
-                        "src",
-                        "reactviews",
-                        "assets",
-                    ),
-                )
-                .toString() + "/";
+
+        const baseUrl = this._getWebview().asWebviewUri(
```

Comment:
Are we using this `baseUrl` somewhere else? Just curious if there's something special about this change.

Reply:
The base URL sets the root of the webview. Any resource referred by the webview will be relative to this base url. We use it to load images within the webviews.